04_CollectionTypes.playground/Contents.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+/*
+ MARK: - 🏠 Домашнее задание.
+ 
+ TODO: - 🧑‍💻 Выполните каждый из пунктов для закрепления материала. Решение пишите на следующей строке после задания:
+ // - проинициализируйте два пустых массива foodArray and drinkArray разными способами
+ */
+//var foodArray: [String] = []
+//var drinkArray: [String] = []
+//взял в кавычки так как выдавало ошибку
+
+ // - присвойте первому массиву стринг значения - названия продуктов (минимум 5)
+var foodArray: [String] = ["burger", "sandwich", "frenchfries", "chocolate", "cheese", "sausage"]
+
+ // - присвойте второму массиву названия напитков (минимум 5)
+var drinkArray: [String] = ["milk", "cola" , "coffee", "beer" , "water", "juice"]
+
+ // - добавьте в конец foodArray еще одно значение
+foodArray.insert("butter", at: 6)
+
+ // - добавьте в середину drinkArray еще одно значение
+drinkArray.insert("coconut milk", at: 3)
```

Comment:
here as well 
what if drinkArray has not 6 items but 8? 
3 won't be a middle index, u need to calculate middle index